wrappers/wrapper.rb

```diff
@@ -1578,6 +1578,8 @@ def simplify_service_setup_duration_and_vehicle_setup_modifiers(vrp, solution =
         overall_total_travel_time_correction = 0
         solution.routes.each{ |route|
           vehicle = vehicles_grouped_by_vehicle_id[route[:vehicle_id]].first
+          vehicle.coef_setup = vehicle[:simplified_coef_setup]
+          vehicle.additional_setup = vehicle[:simplified_additional_setup]
```

Comment:
This is done by `:rewind` logic, do we need it?

Reply:
The `:rewind` is applied once the problem is over. We need to have correct intermediate solutions too

Reply:
I was mainly confused because of the `nil`s but I see that those are gone now.